packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -4968,6 +4968,7 @@ export interface IGroupDividerProps {
     isSelected?: boolean;
     loadingText?: string;
     onGroupHeaderClick?: (group: IGroup) => void;
+    onGroupHeaderKeyUp?: (group: IGroup, ev: React.KeyboardEvent<HTMLElement>) => void;
```

Comment:
These args should be flipped:

```jsx
onGroupHeaderKeyUp: (ev, group)
```
Why: because normal keyup (and click) html event handlers take in "ev" as the first argument, and that's the expectation. It's wrong to put data first. We violate this principle in many places, but we should avoid doing that going forward.

I think `onGroupHeaderClick` is wrong here but fixing it would be a breaking change.